network/gossip.go

```diff
@@ -61,6 +62,8 @@ func (t *Topic) Publish(obj proto.Message) error {
 		return err
 	}
 
+	metrics.SetGauge([]string{networkMetrics, "egress_bytes"}, float32(len(data)))
```

Comment:
I added `SetGauge` here because it might be useful to see how much data was sent in which point in time. To me that tells more then a simple counter of how many total bytes we sent.

Reply:
In this case it's correct because the definiton of Gauge in metrics lib and Prometheus are equivalent, but they lack the Datadog COUNT type, that is what I was talking about, not monotonic counter.